packages/caliper-core/test/common/utils/caliper-utils.js

```diff
@@ -0,0 +1,62 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+const CaliperUtils = require('../../../lib/common/utils/caliper-utils');
+const ConfigUtil = require('../../../lib/common/config/config-util');
+const Constants = require('../../../lib/common/utils/constants');
+
+const chai = require('chai');
+chai.should();
+
+describe('caliper utilities', () => {
+
+
+    describe('Using the static method "augmentUrlWithBasicAuth" to augment base URLs with basic auth', () => {
```

Comment:
would suggest either removing this describe altogether or changing it to
`when augmenting the urls`
We want to avoid putting implementation details into descriptions if possible and just describe the expected behaviour

Reply:
Will do, slowly adjusting to behavioural descriptions 😂